"force_ansible=True" fails when "lsb_release" executable isn't available

Because [`AnsibleRunner.run_module` uses `run_execpt`](https://github.com/philpep/testinfra/blob/b1e369d9d440931553aebf68872ff08cc21ca0ba/testinfra/utils/ansible_runner.py#L206), there is an [uncatched exception](https://github.com/philpep/testinfra/blob/b1e369d9d440931553aebf68872ff08cc21ca0ba/testinfra/modules/systeminfo.py#L54) when `lsb_release` executable isn't available on managed host. Then `/etc/os-release` isn't read and this failure occurs.

Backtrace:
```
>        inventory = os.environ['MOLECULE_INVENTORY_FILE']
>        uri = 'ansible://%s?ansible_inventory=%s&force_ansible=True'
>        my_host = testinfra.get_host(uri % ('test_host', inventory))
>        my_file = my_host.file("/path/to/file")

molecule/resources/tests/test_custom.py:32: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/tmp/molecule-test-py3/lib/python3.7/site-packages/testinfra/host.py:107: in __getattr__
    obj = module_class.get_module(self)
/tmp/molecule-test-py3/lib/python3.7/site-packages/testinfra/modules/base.py:22: in get_module
    klass = cls.get_module_class(_host)
/tmp/molecule-test-py3/lib/python3.7/site-packages/testinfra/modules/file.py:187: in get_module_class
    if host.system_info.type == "linux":
/tmp/molecule-test-py3/lib/python3.7/site-packages/testinfra/modules/systeminfo.py:143: in type
    return self.sysinfo["type"]
/tmp/molecule-test-py3/lib/python3.7/site-packages/testinfra/utils/__init__.py:44: in __get__
    value = obj.__dict__[self.func.__name__] = self.func(obj)
/tmp/molecule-test-py3/lib/python3.7/site-packages/testinfra/modules/systeminfo.py:40: in sysinfo
    sysinfo.update(**self._get_linux_sysinfo())
/tmp/molecule-test-py3/lib/python3.7/site-packages/testinfra/modules/systeminfo.py:54: in _get_linux_sysinfo
    lsb = self.run("lsb_release -a")
/tmp/molecule-test-py3/lib/python3.7/site-packages/testinfra/host.py:71: in run
    return self.backend.run(command, *args, **kwargs)
/tmp/molecule-test-py3/lib/python3.7/site-packages/testinfra/backend/ansible.py:52: in run
    out = self.run_ansible('shell', module_args=command, check=False)
/tmp/molecule-test-py3/lib/python3.7/site-packages/testinfra/backend/ansible.py:60: in run_ansible
    **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <testinfra.utils.ansible_runner.AnsibleRunner object at 0x7f2d3843c7d0>, host = 'application', module_name = 'shell', module_args = 'lsb_release -a', become = False
check = False, kwargs = {}, cmd = 'ansible --tree %s -i %s -m %s --args %s %s'
args = ['/tmp/tmpwjcf8pt9', '/tmp/molecule/testproject/default/inventory/ansible_inventory.yml', 'shell', 'lsb_release -a', 'application'], d = '/tmp/tmpwjcf8pt9'

    def run_module(self, host, module_name, module_args, become=False,
                   check=True, **kwargs):
        cmd, args = 'ansible --tree %s', [None]
        if self.inventory_file:
            cmd += ' -i %s'
            args += [self.inventory_file]
        cmd += ' -m %s'
        args += [module_name]
        if module_args:
            cmd += ' --args %s'
            args += [module_args]
        if become:
            cmd += ' --become'
        if check:
            cmd += ' --check'
        cmd += ' %s'
        args += [host]
        with TemporaryDirectory() as d:
            args[0] = d
>           out = local.run_expect([0, 2], cmd, *args)
E           AssertionError: Unexpected exit code 8 for CommandResult(command=b"ansible --tree /tmp/tmpwjcf8pt9 -i /tmp/molecule/testproject/default/inventory/ansible_inventory.yml -m shell --args 'lsb_release -a' application", exit_status=8, stdout=b'application | FAILED | rc=127 >>\n/bin/sh: 1: lsb_release: not foundnon-zero return code\n', stderr=None)
E           assert 8 in [0, 2]
E            +  where 8 = CommandResult(command=b"ansible --tree /tmp/tmpwjcf8pt9 -i /tmp/molecule/testproject/default/inventory/ansible_inven...s=8, stdout=b'application | FAILED | rc=127 >>\n/bin/sh: 1: lsb_release: not foundnon-zero return code\n', stderr=None).rc

/tmp/molecule-test-py3/lib/python3.7/site-packages/testinfra/utils/ansible_runner.py:206: AssertionError
```

--- a/test/test_backends.py
+++ b/test/test_backends.py
@@ -95,6 +95,19 @@ def test_encoding(host):
         )
 
 
+@pytest.mark.testinfra_hosts(
+    "ansible://debian_stretch?force_ansible=True")
+def test_ansible_any_error_fatal(host):
+    os.environ['ANSIBLE_ANY_ERRORS_FATAL'] = 'True'
+    try:
+        out = host.run("echo out && echo err >&2 && exit 42")
+        assert out.rc == 42
+        assert out.stdout == 'out'
+        assert out.stderr == 'err'
+    finally:
+        del os.environ['ANSIBLE_ANY_ERRORS_FATAL']
+
+
 @pytest.mark.testinfra_hosts(*(USER_HOSTS + SUDO_USER_HOSTS))
 def test_user_connection(host):
     assert host.user().name == "user"

--- a/testinfra/utils/ansible_runner.py
+++ b/testinfra/utils/ansible_runner.py
@@ -203,7 +203,7 @@ class AnsibleRunner(object):
         args += [host]
         with TemporaryDirectory() as d:
             args[0] = d
-            out = local.run_expect([0, 2], cmd, *args)
+            out = local.run_expect([0, 2, 8], cmd, *args)
             files = os.listdir(d)
             if not files and 'skipped' in out.stdout.lower():
                 return {'failed': True, 'skipped': True,

